tests/solutions.test.ts

```diff
@@ -10,7 +10,7 @@ describe('onSolutionSubmission', function () {
   const disregardedUtility = 3456n
   const solver = `0x${'face'.repeat(10)}`
 
-  const timestamp = batchId * 300n + 42n
+  const timestamp = batchId * 300n + 300n + 42n
```

Comment:
nit: 
```suggestion
  const timestamp = (batchId + 1n) * 300n + 42n
```

I think this would make it clearer that its a timestamp in the next batch.